platform-sdk/swirlds-config-benchmark/src/main/java/module-info.java

```diff
@@ -1,6 +1 @@
-module com.swirlds.config.benchmark {
-    requires com.swirlds.config;
-    requires com.swirlds.config.impl;
-    requires jdk.unsupported;
-    requires jdk.zipfs;
-}
+module com.swirlds.config.benchmark {}
```

Comment:
Might be good to double check with some content experts on this.  It seems odd that a benchmark for config would not require config.

Reply:
There is no code in the "main" Module at all. It's empty and thus it does not require anything.

There is only code in "jmh" and in the build.gradle.file there is the dependency to "config2":

from `platform-sdk/swirlds-config-benchmark/build.gradle.kts`:
``` 
jmhModuleInfo {
    requires("com.swirlds.common")
    requires("com.swirlds.config.api")
    requires("jmh.core")
}
```